Program/Program/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Program
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+            Console.WriteLine("PERSON 1:");
```

Comment:
Calculating Bmi and getting all that data of person could be moved into a single method. 2 people- call that twice.